packages/ui-core/src/components/card/card.component.tsx

```diff
@@ -35,11 +37,13 @@ export const Card = styled.div<Props>`
 `;
 
 Card.defaultProps = {
-  border: 'none',
-  borderRadius: '0px',
   backgroundColor: colors.white['500'],
   hasPadding: true,
   hasShadow: true,
+  padding: 20,
+  borderWidth: 20,
```

Comment:
shouldn't it be `0` instead of `20`? by default we don't have any border in card